framework/deproxy_client.py

```diff
@@ -193,11 +193,14 @@ def make_requests(self, requests):
     def make_request(self, request, **kwargs):
         raise NotImplementedError("Not implemented 'make_request()'")
 
-    def send_request(self, request, expected_status_code: Optional[str] = None, timeout=5) -> None:
+    def send_request(
+        self, request=None, expected_status_code: Optional[str] = None, timeout=5
```

Comment:
I think it is a bad idea.
```python3
client.send_request(expected_status_code="200")
```
or
```python3
client.send_request(request=client.create_request(method="GET", headers=[]), expected_status_code="200"
```

Hidden actions should be avoided in tests. First case does not show which request we are sending. I suggest adding simple request to `DeproxyClient` and `DeproxyClientH2` clients:
```python3
@property
def simple_get_request(self) -> deproxy.Request:
    return self.create_request(method="GET", headers=[])
```